modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -4408,11 +4409,13 @@ cp-compare-content-mini-display-context-help=Provides helper methods to retrieve
 cp-content-list-renderer-key=Commerce Product Content List Renderer Key
 cp-definition=Commerce Product
 cp-definition-help=Represents the commerce product definition that is displayed in the virtual product detail.
+cp-definition-id=Represents the ID of commerce product definition that is displayed.
 cp-definition-link-type-configuration-name=Product Relations
 cp-definitions=Commerce Products
 cp-definitions-help=Represents the list of commerce product definitions that is displayed.
 cp-display-layout-configuration-name=CP Display Layout
 cp-friendly-url-configuration-name=Friendly URL
+cp-instance-id=Represents the ID of commerce product instance
```

Comment:
Same here

Reply:
This is not something we can show to the citizen dev, please change it with something readable.